README.md

```diff
@@ -26,10 +26,9 @@ pip install aqtinstall
 aqt install --outputdir C:\Qt 5.15.2 windows desktop win64_msvc2019_64
 ```
 
-Use vcpkg to install boost, tiff, glm. Make sure the vcpkg target triplet is x64-windows.
-
+Use vcpkg to install the following. Make sure the vcpkg target triplet is x64-windows.
```

Comment:
nit: 
"Make sure the vcpkg target triplet is x64-windows, and install the following:"